index.html

```diff
@@ -11,17 +11,18 @@
     <title>Microverse Porfolio</title>
 </head>
 <body>
-    <div class="nav">
+    <nav class="nav">
         <div class="navigation">
-            <div class="logo">Didier Ganthier</div>
+            <a class="logo" href="#">Didier Ganthier</a>
```

Comment:
- [ ] [OPTIONAL] Please consider  replacing `<div class="navigation">`  with a `nav` tag and as a result use `header` tag as the top level wrapper of this section. this will greatly improve your site semantics and accessibility score.  [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. Essentially it should be structured like this;

```
<header class"your className">
    <nav>
        <a class="logo" href="#">Didier Ganthier</a>
        ....other elements left untouched.
    </nav>
</header>
```